### PR TITLE
[DO NOT MERGE] feat: codecv1 script

### PR DESCRIPTION
**Pull request by colinlyguo**
### Purpose or design rationale of this PR

A script for fetching batch metadata.

Usage: go to the dir of `main.go`, setting the DSN as an env variable.
```
go run main.go 111 (batch index)
```

### CODE REVIEW

**Comment by HAOYUatHZ** [comment]
add to testnet/core/contracts/scripts/deploy.sh
```
echo "Deploying Hash contracts..."
OUTPUT=$(forge script scripts/foundry/DeployHash.s.sol:DeployHash --rpc-url "$SCROLL_L2_DEPLOYMENT_RPC" --broadcast 2>&1)
success_or_exit
L2_HASH_ADDRESSES=$(echo "$OUTPUT" | grep -Eo "(L1|L2)_.*")

echo "Deploying Ecc contracts..."
OUTPUT=$(forge script scripts/foundry/DeployEcc.s.sol:DeployEcc --rpc-url "$SCROLL_L2_DEPLOYMENT_RPC" --broadcast 2>&1)
success_or_exit
L2_ECC_ADDRESSES=$(echo "$OUTPUT" | grep -Eo "(L1|L2)_.*")
```

File: contracts/scripts/relay-skipped-tx.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { ethers } from "hardhat";
+
+dotenv.config();
+
+async function main() {
+  const [deployer] = await ethers.getSigners();
+
+  const l1ScrollMessengerAddress = process.env.L1_SCROLL_MESSENGER_PROXY_ADDR!;
+  const l2EccContractAddress = process.env.L2_ECC_ADDR!;
+  const payload = process.env.SKIPPED_TX_PAYLOAD!; // TODO: calc the payload, parse as bytes
```
Comment (0xmountaintop) [inline contracts/scripts/relay-skipped-tx.ts:13]: can use `0x459be2920000000000000000000000000000000000000000000000000000000000000005` for payload

Comment (0xmountaintop) [reply contracts/scripts/relay-skipped-tx.ts:13]:
```
import {
  getBytes,
} from "ethers";

...
const payloadStr = process.env.SKIPPED_TX_PAYLOAD!
const payload = getBytes(payloadStr);
...
```

Comment (colinlyguo) [reply contracts/scripts/relay-skipped-tx.ts:13]: cast keccak "ecPairings(uint256)"

**Pull request closed without merging**